wire-ios/Wire-iOS Tests/ExtensionSettingsTests.swift

```diff
@@ -37,18 +37,14 @@ final class ExtensionSettingsTests: XCTestCase {
         super.tearDown()
     }
 
-    func testThatItDisablesAnalyticsReportByDefault() {
-        XCTAssertTrue(settings.disableAnalyticsSharing)
-    }
-
     func testThatItHandlesAnalyticsPreferenceChange() {
-        XCTAssertTrue(settings.disableAnalyticsSharing)
+        XCTAssertNil(settings.disableAnalyticsSharing)
 
         settings.disableAnalyticsSharing = false
-        XCTAssertFalse(settings.disableAnalyticsSharing)
+        XCTAssertEqual(settings.disableAnalyticsSharing, false)
 
         settings.disableAnalyticsSharing = true
-        XCTAssertTrue(settings.disableAnalyticsSharing)
+        XCTAssertEqual(settings.disableAnalyticsSharing, true)
```

Comment:
Why did you change this? Just a style change?

Reply:
Because it became optional

Reply:
🤦 of course!